packages/wallet-sdk/src/util/checkCrossOriginOpenerPolicy.test.ts

```diff
@@ -1,17 +1,17 @@
-import { checkCrossOriginOpenerPolicy } from './crossOriginOpenerPolicy';
+import { checkCrossOriginOpenerPolicy } from './checkCrossOriginOpenerPolicy';
 
 describe('checkCrossOriginOpenerPolicy', () => {
   beforeEach(() => {
     // Clear all mocks before each test
     jest.clearAllMocks();
   });
 
-  it('should not run if window is undefined', () => {
+  it('should return true if window is undefined', async () => {
```

Comment:
Update test names to not have `return true` and `return false`, since the checkCrossOriginOpenerPolicy function now returns the actual policy